benchmarks/models/fs_wmt.sh

```diff
@@ -9,18 +9,18 @@
 source utils.sh
 
 # MODEL - wmt16
-./benchmark.sh \
-    fairseq \
-    wmt16.en.de.32k \
-    wmt16_en_de_bpe32k/bin \
-    valid \
-    256
+# ./benchmark.sh \
```

Comment:
This is the baseline. I guess you commented them out for debugging?

Reply:
Yes. I was/am having some trouble getting the benchmarks to work properly, so that is a work in progress.